build.sbt

```diff
@@ -17,6 +17,9 @@ lazy val session = (project in file("session"))
     OsgiKeys.importPackage := Seq(akkaImport(), optionalImport("org.apache.cassandra.*"), "*"),
     OsgiKeys.privatePackage := Nil)
 
+lazy val dumpSchema = taskKey[Unit]("Dumps cassandra schema for docs")
+dumpSchema := (core / runMain in (Test)).toTask(" akka.persistence.cassandra.PrintCreateStatements").value
```

Comment:
Neat but will run it for every build of the docs, perhaps that is ok, not sure there is a super easy way to use the sbt caching infra in a task though.

Reply:
I think that is okay for now